sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_encode.py

```diff
@@ -618,8 +631,7 @@ def encode_fields(value):
     return fields
 
 
-def encode_annotations(value):
-    # type: (Optional[Dict[str, Any]]) -> Dict[str, Any]
+def encode_annotations(value: Optional[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
should this be 
```suggestion
def encode_annotations(value: Optional[Dict[Union[str, bytes], Any]]) -> Dict[Union[str, bytes], Any]:
```

Reply:
good catch, the value type can be str or bytes, but the return is str due to definition of fields being str, Any